introduction.md

```diff
@@ -4,3 +4,6 @@ dojo_rules.md
 introduction.md
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?